src/pages/instant-observability.js

```diff
@@ -18,10 +18,12 @@ import { navigate } from '@reach/router';
 
 import BUILD_YOUR_OWN from '../images/build-your-own.svg';
 import { useDebounce } from 'react-use';
+import { sortFeaturedPacks } from '../utils/sortFeaturedPacks';
 
 const { QUICKSTARTS_REPO } = require('../data/constants');
 
 const FILTERS = [
+  { name: 'Featured', type: 'featured', icon: 'nr-relicans' },
```

Comment:
I _think_ we probably want to handle keyword filters differently. This array (which could probably be renamed) is for filtering by _content_.

If we add `Featured` to this array, it means we wont be able to filter by featured quickstarts _and_ ones that include dashboards. We should probably have a separate list of keyword filters with their own function(s) to narrow down the list. I'm sure @aswanson-nr has some thoughts here too.